src/zkevm_specs/evm/main.py

```diff
@@ -36,8 +36,13 @@ def verify_steps(
                 )
             )
         except AssertionError as e:
-            ok = False
-    assert ok == success
+            exception = e
```

Comment:
I made this change so that when tests fail with `success = True`, the exception is raised so that it's presented to the user (instead of being captured and just seeing an `AssertionError`)